charts/hlf-k8s/templates/deployment-chaincode-operator.yaml

```diff
@@ -163,23 +163,19 @@ spec:
                     --cafile /var/hyperledger/tls/ord/cert/cacert.pem \
                     --keyfile /var/hyperledger/tls/client/pair/tls.key \
                     --certfile /var/hyperledger/tls/client/pair/tls.crt \
-                    -o {{ index $.Values "hlf-ord" "host" }}:{{ index $.Values "hlf-ord" "port" }} > chaincode.approve 2>&1
+                    -o {{ index $.Values "hlf-ord" "host" }}:{{ index $.Values "hlf-ord" "port" }} > chaincode_{{.name}}_{{.version}}.approve 2>&1
 
-
-                  if grep "${CHAINCODE_CCID}" chaincode.approve > /dev/null; then
-                    break
+                  if ! grep -q "${CHAINCODE_CCID}" chaincode_{{.name}}_{{.version}}.approve; then
+                    printf "[DEBUG] Chaincode {{ .name }} {{ .version }} with ccid ${CHAINCODE_CCID} is not approved on channel {{ $channel.channelname }}\n"
+                    sleep 5
                   fi
 
-                  printf "[DEBUG] Chaincode {{ .name }} {{ .version }} with ccid ${CHAINCODE_CCID} is not approved on channel {{ $channel.channelname }}\n"
-
-                  sleep 5
-
                 done
 
 
                 ## Check chaincode commit readiness
 
-                while true; do
+                until grep -q "true" chaincode.{{ .name }}.ready; do
```

Comment:
I think the `version` should also appear here, unless I'm missing a reason to omit it?